examples/NSIDC/ISat2_from_CMR_to_S3.ipynb

```diff
@@ -0,0 +1,665 @@
+       "        Size(MB): 877.0574979782\n",
+       "        Data: ['https://n5eil01u.ecs.nsidc.org/DP9/ATLAS/ATL03.004/2018.10.18/ATL03_20181018000228_02960102_004_01.h5']\n",
+       "\n",
+       "        "
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# now that we have the concept-ids we can look for data granules in that collection and pass spatiotemporal parameters.\n",
+    "from cmr_serializer import QueryResult\n",
+    "\n",
+    "# Jeneau ice sheet\n",
+    "query = {'concept-id': 'C1997321091-NSIDC_ECS',\n",
+    "         'bounding_box': '-135.1977,58.3325,-133.3410,58.9839'}\n",
+    "\n",
+    "# Querying for ATL03 v3 using its concept-id and a bounding box\n",
+    "results = granule.search(query, limit=1000)\n",
+    "# This is a wrapper with convenient methods to work with CMR query results.\n",
+    "granules = QueryResult(results).items()\n",
+    "\n",
+    "print(f\"Total granules found: {len(results)} \\n\")\n",
+    "for g in granules[0:3]:\n",
+    "    display(g)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "041d4fab-e190-49e9-a9e7-36905bae01b8",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "['https://n5eil01u.ecs.nsidc.org/DP9/ATLAS/ATL03.004/2018.10.14/ATL03_20181014001049_02350102_004_01.h5']\n",
+      "['https://n5eil01u.ecs.nsidc.org/DP9/ATLAS/ATL03.004/2018.10.15/ATL03_20181015124359_02580106_004_01.h5']\n",
+      "['https://n5eil01u.ecs.nsidc.org/DP9/ATLAS/ATL03.004/2018.10.18/ATL03_20181018000228_02960102_004_01.h5']\n",
+      "['https://n5eil01u.ecs.nsidc.org/DP9/ATLAS/ATL03.004/2018.11.05/ATL03_20181105113651_05780106_004_01.h5']\n",
+      "['https://n5eil01u.ecs.nsidc.org/DP9/ATLAS/ATL03.004/2018.11.07/ATL03_20181107225525_06160102_004_01.h5']\n",
+      "['https://n5eil01u.ecs.nsidc.org/DP9/ATLAS/ATL03.004/2018.11.09/ATL03_20181109112837_06390106_004_01.h5']\n",
+      "['https://n5eil01u.ecs.nsidc.org/DP9/ATLAS/ATL03.004/2018.11.11/ATL03_20181111224708_06770102_004_01.h5']\n",
+      "['https://n5eil01u.ecs.nsidc.org/DP9/ATLAS/ATL03.004/2018.11.15/ATL03_20181115223845_07380102_004_01.h5']\n",
+      "['https://n5eil01u.ecs.nsidc.org/DP9/ATLAS/ATL03.004/2018.12.04/ATL03_20181204101243_10200106_004_01.h5']\n",
+      "['https://n5eil01u.ecs.nsidc.org/DP9/ATLAS/ATL03.004/2018.12.06/ATL03_20181206213114_10580102_004_01.h5']\n"
+     ]
+    }
+   ],
+   "source": [
+    "# We  can access the data links with the data_links()\n",
+    "for g in granules[0:10]:\n",
+    "    print(g.data_links())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "laughing-malawi",
+   "metadata": {},
+   "source": [
+    "## Cloud Collections\n",
+    "\n",
+    "Since NSIDC cloud collections are not yet public we need to authenticate ourselves with CMR first.\n"
```

Comment:
Question: are you going to be publishing this quarto book and using it to demonstrate accessing non-public datasets? Or do you plan to remove this piece before the book is officially public?

Reply:
I have the same question, I don't really know why these datasets are not public. Even if they are preliminary and partial. Maybe @asteiker knows the administrative reason. The other oddity is that the same data is at least visible using the STAC catalog via `cmr-stac`.

Reply:
The short answer is that we only recently ingested these data to production, so there are lot of other pieces that need to be in place before we fully, operationally support the collection publicly in the cloud. That's interesting about cmr-stac... I would have assumed that the same ACL's control the visibility of those collections in CMR. I wonder if this is a better option to focus on while the data are still restricted since at least then we don't have to describe the need for a CMR token, etc.